docs/pages/add-token.mdx

```diff
@@ -0,0 +1,24 @@
+# Create New Token on Existing Collection
+
+This guide will walk you through the process of creating a new token on an existing InProcess collection.
+
+## Steps
+
+1. Navigate to the Vercel page at [https://setup-actions.vercel.app/](https://setup-actions.vercel.app/)
+2. On the Vercel page:
+   - Uncheck all boxes except for `SetupNewToken`
+   - Enter the tokenID for the next token
+   - Enter the funds recipient
+   - Enter the address for Fixed Price: `0x2994762aA0E4C750c51f333C10d81961faEBE785`
+3. Copy the generated bytes text
+4. Go to your contract on Etherscan
+5. Find function `#9 - Multicall` in the Contract -> `Write as Proxy' section
+6. Paste the generated bytes text
+7. Click `Write` to start the transaction
```

Comment:
could you please add screenshots here to help show, visually, how these steps work? we're getting pretty technical and I want to make sure non-techies can follow these instructions.